conda/base/context.py

```diff
@@ -1165,6 +1171,48 @@ def cpu_flags(self):
         info = _get_cpu_info()
         return info["flags"]
 
+    @memoizedproperty
+    @unique_sequence_map(
+        unique_key="backend",
+        property_name="_reporters",
+        allowed_keys={"json", "stdlib"},
+    )
+    def reporters(self) -> tuple[Mapping[str, str]]:
+        """
+        Determine the value of reporters based on other settings and the ``self._reporters``
+        value itself.
+        """
+        if context.json:
+            return (
+                {
+                    "backend": "json",
+                    "output": "stdout",
+                    "verbosity": self.verbosity,
+                },
+            )
+
+        if context.quiet:
+            return (
+                {
+                    "backend": "stdlib",
+                    "output": "stdout",
+                    "verbosity": self.verbosity,
+                    "quiet": True,
+                },
+            )
```

Comment:
I'm not convinced this is the correct thing to do for `context.quiet`. I'm wondering if quiet should be inserted into whatever the reporters value.

It's also harder to reason about this since the reporters aren't in use yet so maybe just leave a comment and revisit once the next PR is opened?

Reply:
I think your correct on that. I will just insert `context.quiet` as a value to this dictionary.

Reply:
I have vastly simplified this method in a recent commit. Please check it out and let me know if there are still any concerns.